doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -60,26 +60,25 @@ X509_STORE_CTX_new() is the same as X509_STORE_CTX_new_ex() except that
 the default library context and a NULL property query string are used.
 
 X509_STORE_CTX_cleanup() internally cleans up an B<X509_STORE_CTX> structure.
-The context can then be reused with a new call to X509_STORE_CTX_init().
+It is used by X509_STORE_CTX_init() and X509_STORE_CTX_free().
 
 X509_STORE_CTX_free() completely frees up I<ctx>. After this call I<ctx>
 is no longer valid.
 If I<ctx> is NULL nothing is done.
 
 X509_STORE_CTX_init() sets up I<ctx> for a subsequent verification operation.
-It must be called before each call to L<X509_verify_cert(3)>, i.e., a I<ctx> is
-only good for one verification; if you want to verify a second certificate
-or chain with the same I<ctx> then you must call X509_STORE_CTX_cleanup()
-and then X509_STORE_CTX_init() again before the second call to
-L<X509_verify_cert(3)> or L<X509_STORE_CTX_verify(3)>.
+It must be called before each call to L<X509_verify_cert(3)> or
+L<X509_STORE_CTX_verify(3)>, i.e., a I<ctx> is only good for one verification.
```

Comment:
`an I<ctx>`

Reply:
Well, the text will read "an _ctx_" where 'c' is a consonant, but done anyway.

Reply:
In English a/an is determined by the *sound* of what follows.  So "a ctx" makes more sense, although I suppose there's some way to pronounce `ctx` that has an initial vowel sound (spelling the letters out doesn't do it).

Reply:
Thanks @richsalz for the confirmation.
I suppose @t8m was confused by the `I` which is just (hidden) markup here.
I've avoided the issue for now by using "a context" :smiley:

Reply:
yeah, I have no idea why I suggested this. Probably got confused by the `I`.